osu.Game/Screens/Edit/Editor.cs

```diff
@@ -235,16 +237,27 @@ public override void OnEntering(IScreen last)
             Beatmap.Value.Track?.Stop();
         }
 
+        private bool isExitingGame = false;
+
         public override bool OnExiting(IScreen next)
         {
             Background.FadeColour(Color4.White, 500);
 
             if (Beatmap.Value.Track != null)
```

Comment:
don't you think using a local variable is better instead of accessing the track property every time

Reply:
The line `if (Beatmap.Value.Track != null)` was already here before I started making any changes. I felt safer leaving it as it was. Besides, it just runs once, while we're exiting. Would it really boost perforamnce?

Reply:
it's less performance and more code quality